### PR TITLE
PEP8 fixes. Some optimizations

### PR DESCRIPTION
**Pull request by Danfocus**

### CODE REVIEW

File: Contents/DefaultPrefs.json [hunk 1]
```diff
@@ -13,99 +13,20 @@
             "prem1",
             "prem2",
             "prem3",
-            "pub1",
+            "prem4"
         ],
-        "default": "pub1"
+        "default": "prem1"
```
Comment (phrawzty) [inline Contents/DefaultPrefs.json]: Yeah good call given that the public stream doesn't exist anymore. :P

File: Contents/Code/audioaddict.py [hunk 2]
```diff
@@ -20,75 +21,19 @@ def __init__(self):
...
-                'public1':                  {'codec': 'aac', 'bitrate':  40},
-                'public3':                  {'codec': 'mp3', 'bitrate':  64},
-                'premium_low':              {'codec': 'aac', 'bitrate':  40},
-                'premium_medium':           {'codec': 'aac', 'bitrate':  64},
-                'premium':                  {'codec': 'aac', 'bitrate': 128},
-                'premium_high':             {'codec': 'mp3', 'bitrate': 256}
-            }
+            'premium_medium':           {'codec': 'aac', 'bitrate':  64},
+            'premium':                  {'codec': 'aac', 'bitrate': 128},
+            'premium_high':             {'codec': 'mp3', 'bitrate': 320},
```
[the remark this answers is not in the document]
Comment (Danfocus) [reply Contents/Code/audioaddict.py]: You can check this (mobile/apps)
https://api.audioaddict.com/v1/di/qualities?api_key=YOUR_API_KEY
returns
`[{
	"id": 3,
	"key": "high",
	"name": "High",
	"position": 2,
	"premium_only": true,
	"default": true,
	"content_format": {
		"extension": "mp4",
		"id": 3,
		"key": "aac",
		"mime_type": "audio/mp4",
		"name": "AAC"
	},
	"content_quality": {
		"id": 4,
		"key": "128k",
		"kilo_bitrate": 128,
		"name": "128 kbit/s"
	}
}, {
	"id": 2,
	"key": "medium",
	"name": "Medium",
	"position": 1,
	"premium_only": false,
	"default": false,
	"content_format": {
		"extension": "mp4",
		"id": 6,
		"key": "aac_he",
		"mime_type": "audio/mp4",
		"name": "AAC-HE"
	},
	"content_quality": {
		"id": 2,
		"key": "64k",
		"kilo_bitrate": 64,
		"name": "64 kbit/s"
	}
}, {
	"id": 4,
	"key": "ultra",
	"name": "Ultra",
	"position": 3,
	"premium_only": true,
	"default": false,
	"content_format": {
		"extension": "mp3",
		"id": 1,
		"key": "mp3",
		"mime_type": "audio/mpeg",
		"name": "MP3"
	},
	"content_quality": {
		"id": 6,
		"key": "320k",
		"kilo_bitrate": 320,
		"name": "320 kbit/s"
	}
}]`
other servers returns the same.

File: Contents/DefaultPrefs.json [hunk 3]
```diff
@@ -13,99 +13,20 @@
...
     },
     {
-        "id": "stream_pref_di",
+        "id": "stream_pref",
```
[the remark this answers is not in the document]
Comment (Danfocus) [reply Contents/DefaultPrefs.json]: Answered above

File: Contents/DefaultPrefs.json (hunk 1 above)
Comment (Danfocus) [reply Contents/DefaultPrefs.json]: I observe that prem1 exists in all returned playlists.

**Comment by Danfocus** [comment]
Main optimization is removing get_streamurl call

**Comment by phrawzty** [comment]
> Main optimization is removing get_streamurl call

Indeed! Well it all looks reasonable to me, hehe. Thanks for the PR!

**Pull request merged**